sm8250.c

```diff
@@ -0,0 +1,493 @@
+	{ "measure_only_gpu_cc_cx_gfx3d_slv_clk", &gcc, 0x163, &gpu_cc, 0x1B },
+	{ "measure_only_gpu_cc_gx_gfx3d_clk", &gcc, 0x163, &gpu_cc, 0xB },
+
+	{ "npu_cc_atb_clk", &gcc, 0x180, &npu_cc, 0x17 },
+	{ "npu_cc_bto_core_clk", &gcc, 0x180, &npu_cc, 0x19 },
+	{ "npu_cc_bwmon_clk", &gcc, 0x180, &npu_cc, 0x18 },
+	{ "npu_cc_cal_hm0_cdc_clk", &gcc, 0x180, &npu_cc, 0xB },
+	{ "npu_cc_cal_hm0_clk", &gcc, 0x180, &npu_cc, 0x2 },
+	{ "npu_cc_cal_hm0_dpm_ip_clk", &gcc, 0x180, &npu_cc, 0xC },
+	{ "npu_cc_cal_hm0_perf_cnt_clk", &gcc, 0x180, &npu_cc, 0xD },
+	{ "npu_cc_cal_hm1_cdc_clk", &gcc, 0x180, &npu_cc, 0xE },
+	{ "npu_cc_cal_hm1_clk", &gcc, 0x180, &npu_cc, 0x3 },
+	{ "npu_cc_cal_hm1_dpm_ip_clk", &gcc, 0x180, &npu_cc, 0xF },
+	{ "npu_cc_cal_hm1_perf_cnt_clk", &gcc, 0x180, &npu_cc, 0x10 },
+	{ "npu_cc_core_clk", &gcc, 0x180, &npu_cc, 0x4 },
+	{ "npu_cc_dl_dpm_clk", &gcc, 0x180, &npu_cc, 0x23 },
+	{ "npu_cc_dl_llm_clk", &gcc, 0x180, &npu_cc, 0x22 },
+	{ "npu_cc_dpm_clk", &gcc, 0x180, &npu_cc, 0x8 },
+	{ "npu_cc_dpm_temp_clk", &gcc, 0x180, &npu_cc, 0x14 },
+	{ "npu_cc_dpm_xo_clk", &gcc, 0x180, &npu_cc, 0xA },
+	{ "npu_cc_dsp_ahbm_clk", &gcc, 0x180, &npu_cc, 0x1C },
+	{ "npu_cc_dsp_ahbs_clk", &gcc, 0x180, &npu_cc, 0x1B },
+	{ "npu_cc_dsp_axi_clk", &gcc, 0x180, &npu_cc, 0x1E },
+	{ "npu_cc_dsp_bwmon_ahb_clk", &gcc, 0x180, &npu_cc, 0x1D },
+	{ "npu_cc_dsp_bwmon_clk", &gcc, 0x180, &npu_cc, 0x1F },
+	{ "npu_cc_isense_clk", &gcc, 0x180, &npu_cc, 0x7 },
+	{ "npu_cc_llm_clk", &gcc, 0x180, &npu_cc, 0x6 },
+	{ "npu_cc_llm_curr_clk", &gcc, 0x180, &npu_cc, 0x21 },
+	{ "npu_cc_llm_temp_clk", &gcc, 0x180, &npu_cc, 0x15 },
+	{ "npu_cc_llm_xo_clk", &gcc, 0x180, &npu_cc, 0x9 },
+	{ "npu_cc_noc_ahb_clk", &gcc, 0x180, &npu_cc, 0x13 },
+	{ "npu_cc_noc_axi_clk", &gcc, 0x180, &npu_cc, 0x12 },
+	{ "npu_cc_noc_dma_clk", &gcc, 0x180, &npu_cc, 0x11 },
+	{ "npu_cc_rsc_xo_clk", &gcc, 0x180, &npu_cc, 0x1A },
+	{ "npu_cc_s2p_clk", &gcc, 0x180, &npu_cc, 0x16 },
+	{ "npu_cc_xo_clk", &gcc, 0x180, &npu_cc, 0x1 },
+
+	{ "video_cc_ahb_clk", &gcc, 0x57, &video_cc, 0x7 },
+	{ "video_cc_mvs0_clk", &gcc, 0x57, &video_cc, 0x3 },
+	{ "video_cc_mvs0c_clk", &gcc, 0x57, &video_cc, 0x1 },
+	{ "video_cc_mvs1_clk", &gcc, 0x57, &video_cc, 0x5 },
+	{ "video_cc_mvs1_div2_clk", &gcc, 0x57, &video_cc, 0x8 },
+	{ "video_cc_mvs1c_clk", &gcc, 0x57, &video_cc, 0x9 },
+	{ "video_cc_sleep_clk", &gcc, 0x57, &video_cc, 0xC },
+	{ "video_cc_xo_clk", &gcc, 0x57, &video_cc, 0xB },
+
+	// { "measure_only_mccc_clk", &gcc, 0xd1, &mc_cc },
+
+	{ "measure_only_apcs_gold_post_acd_clk", &gcc, 0xe7, &apss_cc, 0x25, /* TODO: Are these pre_div_vals? */ 8 },
+	{ "measure_only_apcs_goldplus_post_acd_clk", &gcc, 0xe7, &apss_cc, 0x61, /* TODO: Are these pre_div_vals? */ 8 },
+	{ "measure_only_apcs_l3_post_acd_clk", &gcc, 0xe7, &apss_cc, 0x41, /* TODO: Are these pre_div_vals? */ 4 },
+	{ "measure_only_apcs_silver_post_acd_clk", &gcc, 0xe7, &apss_cc, 0x21, /* TODO: Are these pre_div_vals? */ 4 },
```

Comment:
Not really certain if these are correct, random readback from my device:

```
               measure_only_apcs_gold_post_acd_clk: 338.750448MHz (338750448Hz)
           measure_only_apcs_goldplus_post_acd_clk: 2744.580960MHz (2744580960Hz)
                 measure_only_apcs_l3_post_acd_clk: 1612.086072MHz (1612086072Hz)
             measure_only_apcs_silver_post_acd_clk: 1315.384920MHz (1315384920Hz)
```

Reply:
these should be 1:1 with what cpufreq reports, as it reads back the LMh-throttled values